custom_dc/customize_ui.md

```diff
@@ -21,9 +21,9 @@ instructions](https://github.com/datacommonsorg/website#github-workflow) into a
 new repo, which will be used as the custom Data Commons codebase. Custom Data
 Commons development and deployment will be based on this forked repo.
 
-To test codes in a local environment, follow this
+To run website in a local environment (Mac, Unix), follow this
```

Comment:
nit: s/Unix/Linux/?  We don't support other unixes right

Reply:
Done